### PR TITLE
Refactored AverageRating Component to work with new props

### PR DESCRIPTION
**Pull request by UnlikelyHero**
... and made it easier to implement...

this will break other parent components until the productMeta prop is provided... view the readme comment for details

### CODE REVIEW

File: client/src/components/AverageRating.jsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import axios from 'axios';
 import Rating from '@material-ui/lab/Rating';
 
 const getRatingInfo = (ratings) => {
+  if (ratings.length === 0) return 0;
```
Comment (bkern98) [inline client/src/components/AverageRating.jsx]: Is it right to return 0 or should you be returning an object with average: 0 and total: 0?

**Review by bkern98 — COMMENTED** [review]
Just have the one comment regarding the case with no ratings

File: client/src/components/AverageRating.jsx (hunk 1 above)
Comment (UnlikelyHero) [reply client/src/components/AverageRating.jsx]: nope! thanks

Comment (UnlikelyHero) [reply client/src/components/AverageRating.jsx]: I've removed that logic, it is incorrect

**Review by bkern98 — APPROVED** [review]

**Comment by UnlikelyHero** [comment]
Before merging, I'm going to make a few more corrections. we'll close this PR for now

**Pull request merged**